frontend/src/utils/timeUtil.ts

```diff
@@ -75,3 +75,17 @@ export const getAllSameDays = (day: number, begin: Date, end: Date) => {
 
   return dates;
 };
+
+export const getTimestampDiff = (
+  aheadTimestamp: number,
+  behindTimestamp: number
+) => {
+  if (aheadTimestamp <= behindTimestamp) {
+    return '00:00';
+  }
+
+  const minute = Math.floor((aheadTimestamp - behindTimestamp) / 1000 / 60);
+  const second = Math.floor(((aheadTimestamp - behindTimestamp) / 1000) % 60);
+
+  return `${('0' + minute).slice(-2)}:${('0' + second).slice(-2)}`;
```

Comment:
[String.prototype.padStart](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/String/padStart)로 lpad 쉽게 할 수 있습니다..!

```suggestion
  const minute = Math.floor((aheadTimestamp - behindTimestamp) / 1000 / 60);
  const second = Math.floor(((aheadTimestamp - behindTimestamp) / 1000) % 60);
  const leftPaddedMinute = minute.toString().padStart(2, '0');
  const leftPaddedSecond = second.toString().padStart(2, '0');

  return `${leftPaddedMinute}:${leftPaddedSecond}`;
```

Reply:
오.. 쉬운 방법이 있었네요!  👍